readme.md

```diff
@@ -363,6 +363,10 @@ test(t => {
 
 AVA comes with builtin support for ES2015 through [Babel 6](https://babeljs.io). Just write your tests in ES2015. No extra setup needed. You can use any Babel version in your project. We use our own bundled Babel with the [`es2015`](http://babeljs.io/docs/plugins/preset-es2015/) and [`stage-2`](http://babeljs.io/docs/plugins/preset-stage-2/) presets.
 
+### TypeScript support
+
+AVA includes typings for TypeScript. You have to setup transpilation yourself. When you set `module` to `commonjs`, TypeScript will automatically find the type definitions for AVA. You should set `target` to `es2015` to use Promises and async functions.
```

Comment:
> When you set `module` to `commonjs`,

Maybe mention where this is set? It can easily be confused as being set in the AVA config.